### DISCUSSION TITLE
Change Mouse Sensitivity under 0.1x?

### DISCUSSION THREAD

**Discussion by Bwine127** [ppy/osu#27470] [Q&A]
I tried to change input.json to 0.005, but mouse sensitivity is 0.1x in game. (And it resets to 0.1x when game closes.)
Can I avoid value limit? It did work on osu!classic(osu!user.cfg/MouseSpeed(but it changes to 0.01 when closes)), but not in osu!lazer(input.json).
unfortunately I can't change mouse DPI

**Comment by Rekunan** [comment]
This has been suggested before and denied: #25045

It sounds like a massive issue if you can't change your mouse DPI. What about changing the mouse sensitivity through windows settings?
